plugins/reach/services/EntryVendorTaskService.php

```diff
@@ -78,7 +78,7 @@ public function addEntryVendorTaskImpl($entryVendorTask, $taskVersion, $dbEntry,
 		$existingTask = EntryVendorTaskPeer::retrieveOneActiveOrCompleteTask($entryVendorTask->entryId, $entryVendorTask->catalogItemId, kCurrentContext::getCurrentPartnerId(), $taskVersion);
 		if ($existingTask)
 		{
-			if (!$dbVendorCatalogItem->getAllowResubmission() ||
+			if ((!$dbVendorCatalogItem->getAllowResubmission() && !$entryVendorTask->isScheduled()) ||
```

Comment:
this going to affect all tasks? not just live?

Reply:
No. only the live caption tasks currently return isScheduled true